### PR TITLE
Beatmap Nominators: Recent changes and additions

### PR DESCRIPTION
**Pull request by Naxesss**
**Additions**
**o!t**
Aloda

**o!c**
Hareimu
Sorcerer
Nokashi

**o!m**
Garalulu
Kamikaze
Tofu1222

**Removals**
**o!s**
ezek
MrSergio
Monstrata
Kurai
Bonsai
Rizia
Zero__wind
sheela

**o!m**
Protastic101

**Changes**
Renamed MariahCarey to Voli
Added catch to Sinnoh and celerih
Added french to celerih

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -59,12 +59,13 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
...
+| [Aloda](https://osu.ppy.sh/u/1190127)             | ![Yes][true] | ![Yes][true] | ![No][false] | ![No][false] |                         |
 | [Arrival](https://osu.ppy.sh/u/1694000)           | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | French                   |
 | [Asherz007](https://osu.ppy.sh/u/9014047)         | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
 | [Battle](https://osu.ppy.sh/u/4037545)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Benny-](https://osu.ppy.sh/u/4023183)            | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Norwegian                |
 | [Bonsai](https://osu.ppy.sh/u/987334)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                   |
-| [celerih](https://osu.ppy.sh/u/4696296)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                         |
+| [celerih](https://osu.ppy.sh/u/4696296)           | ![Yes][true] | ![No][false] | ![Yes][false] | ![No][false] | French                 |
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: for the one you changed, change the `[false]` to `[true]`

File: wiki/People/Beatmap_Nominators/en.md [hunk 2]
```diff
@@ -98,35 +101,36 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
...
-| [Protastic101](https://osu.ppy.sh/u/6712747)      | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
 | [Regraz](https://osu.ppy.sh/u/3076909)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Chinese                  |
 | [Rivals_7](https://osu.ppy.sh/u/4610379)          | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Indonesian               |
 | [Rizia](https://osu.ppy.sh/u/1367570)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Cantonese, Chinese       |
 | [sahuang](https://osu.ppy.sh/u/5318910)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Chinese                  |
 | [sheela](https://osu.ppy.sh/u/1138027)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | French                   |
-| [Sinnoh](https://osu.ppy.sh/u/4236057)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                         |
+| [Sinnoh](https://osu.ppy.sh/u/4236057)            | ![Yes][true] | ![No][false] | ![Yes][false] | ![No][false] |                         |
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: ditto

File: wiki/People/Beatmap_Nominators/en.md (hunk 1 above)
Comment (Naxesss) [reply wiki/People/Beatmap_Nominators/en.md]: oops fixed

File: wiki/People/Beatmap_Nominators/en.md [hunk 3]
```diff
@@ -98,35 +101,36 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
...
 | [Lasse](https://osu.ppy.sh/u/896613)              | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                   |
 | [Litharrale](https://osu.ppy.sh/u/3189995)        | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
-| [MariahCarey](https://osu.ppy.sh/u/2522275)       | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [MaridiuS](https://osu.ppy.sh/u/4496961)          | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Serbian                 |
 | [Maxus](https://osu.ppy.sh/u/4335785)             | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Indonesian               |
 | [MBomb](https://osu.ppy.sh/u/3071175)             | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] |                          |
 | [Mir](https://osu.ppy.sh/u/8688812)               | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Monstrata](https://osu.ppy.sh/u/2706438)         | ![Yes][true] | ![Yes][true] | ![Yes][true] | ![No][false] |                          |
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: Put on hold until it's decided if Monstrata stays

File: wiki/People/Beatmap_Nominators/en.md [hunk 4]
```diff
@@ -98,35 +101,36 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
 | [Kurai](https://osu.ppy.sh/u/77089)               | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | French                   |
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: remove this guy

File: wiki/People/Beatmap_Nominators/en.md [hunk 5]
```diff
@@ -59,12 +59,13 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
...
 | [Absolute Zero](https://osu.ppy.sh/u/4847256)     | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] |                          |
 | [alienflybot](https://osu.ppy.sh/u/636114)        | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Cantonese, Chinese       |
+| [Aloda](https://osu.ppy.sh/u/1190127)             | ![Yes][true] | ![Yes][true] | ![No][false] | ![No][false] |                         |
 | [Arrival](https://osu.ppy.sh/u/1694000)           | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | French                   |
 | [Asherz007](https://osu.ppy.sh/u/9014047)         | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
 | [Battle](https://osu.ppy.sh/u/4037545)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Benny-](https://osu.ppy.sh/u/4023183)            | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Norwegian                |
 | [Bonsai](https://osu.ppy.sh/u/987334)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                   |
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: bonsai :(

**Pull request merged**